### PR TITLE
Portuguese Translations

### PR DESCRIPTION
**Pull request by mclemente**
Added Portuguese translations to Chat Enhancements and Curvy Walls.
Updated Scene Enhancement's Portuguese translation.
Both translations are from @Brn086 who doesn't know how to Github.

### CODE REVIEW

File: df-chat-enhance/module.json [hunk 1]
```diff
@@ -1,32 +1,57 @@
...
+		"css/adventure-log.css",
+		"css/archive-manager.css",
+		"css/archive-new.css",
+		"css/chat-editor.css",
+		"css/privacy-button.css"
+	],
 	"languages": [
-		{ "lang": "en", "name": "English", "path": "lang/en.json" }
+		{ "lang": "en", "name": "English", "path": "lang/en.json" },
+		{ "lang": "pt-BR", "name": "Português (Brasil)", "path": "lang/pt-BR.json"}
```
Comment (flamewave000) [inline df-chat-enhance/module.json:42]: Please remove all other changes in the `df-chat-enhance/module.json` file except for these 2 lines in the languages part.

File: df-curvy-walls/module.json [hunk 2]
```diff
@@ -1,30 +1,44 @@
...
+	"styles": [
+		"css/segment-controls.css"
+	],
 	"languages": [
 		{ "lang": "en", "path": "lang/en.json", "name": "English" },
 		{ "lang": "fr", "path": "lang/fr.json", "name": "French" },
 		{ "lang": "es", "path": "lang/es.json", "name": "Español" },
-		{ "lang": "ja", "path": "lang/ja.json", "name": "日本語" }
+		{ "lang": "ja", "path": "lang/ja.json", "name": "日本語" },
+		{ "lang": "pt-BR", "path": "lang/pt-BR.json", "name": "Português (Brasil)"}
```
Comment (flamewave000) [inline df-curvy-walls/module.json:30]: Please remove all other changes in the `df-curvy-walls/module.json` file except for these two lines in the languages part.

**Review by flamewave000 — CHANGES_REQUESTED** [review]
Thank you so much for the localisations. I've left 2 comments for some minor changes. The module.json files are templated, and the templating is being removed in this PR. I would like you to keep the languages update in the file, but revert all the other changes in those 2 files.

**Comment by mclemente** [comment]
Ok, done. I wasn't sure about that logo media, so I added it back into the file just in case.

**Review by flamewave000 — APPROVED** [review]

**Comment by flamewave000** [comment]
Looks good! Thank you :)

**Pull request merged**